src/generator/LevelGenerator.cpp

```diff
@@ -181,6 +186,29 @@ LevelGenerator::Generate() {
           }
           objs->push_back(std::make_shared<gameobject::Platform>(
               glm::vec3(xPos, yPos, zPos), glm::vec3(power, .5f, 7.0f)));
+          if ((rand() % 2) + 1 == 1) {
```

Comment:
I might be late saying this if there are already uses of rand() in the level generator, but I don't think that you should get a different level each time you play the same song